__tests__/api.test.js

```diff
@@ -17,7 +17,7 @@ describe("API endpoints", () => {
         .get("/api/topics")
         .expect(200)
             .then((result) => {
-                const topics = result.body;
+                const topics = result.body.topics;
```

Comment:
Great work, but the test assertion below is only checking the first element which isnt very robust. I would recommend testing the whole array to ensure every element in the topics array looks how you expect it to.

Reply:
Thank you, I replaced the test with whole array checking.